src/path_info.cpp

```diff
@@ -178,51 +108,19 @@ void PATH_INFO::set_standard_filenames()
-    update_pathname( "config_dir", dir );
+    config_dir_value = dir;
 #else
-    update_pathname( "config_dir", FILENAMES["user_dir"] + "config/" );
-#endif
-    update_pathname( "graveyarddir", FILENAMES["user_dir"] + "graveyard/" );
-
-    update_pathname( "options", FILENAMES["config_dir"] + "options.json" );
-    update_pathname( "panel_options", FILENAMES["config_dir"] + "panel_options.json" );
-    update_pathname( "keymap", FILENAMES["config_dir"] + "keymap.txt" );
-    update_pathname( "user_keybindings", FILENAMES["config_dir"] + "keybindings.json" );
-    update_pathname( "debug", FILENAMES["config_dir"] + "debug.log" );
-    update_pathname( "crash", FILENAMES["config_dir"] + "crash.log" );
-    update_pathname( "fontlist", FILENAMES["config_dir"] + "fontlist.txt" );
-    update_pathname( "fontdata", FILENAMES["config_dir"] + "fonts.json" );
-    update_pathname( "autopickup", FILENAMES["config_dir"] + "auto_pickup.json" );
-    update_pathname( "safemode", FILENAMES["config_dir"] + "safemode.json" );
-    update_pathname( "base_colors", FILENAMES["config_dir"] + "base_colors.json" );
-    update_pathname( "custom_colors", FILENAMES["config_dir"] + "custom_colors.json" );
-    update_pathname( "mods-user-default", FILENAMES["config_dir"] + "user-default-mods.json" );
-    update_pathname( "lastworld", FILENAMES["config_dir"] + "lastworld.json" );
-    update_pathname( "user_moddir", FILENAMES["user_dir"] + "mods/" );
-    update_pathname( "worldoptions", "worldoptions.json" );
-
-    // Needed to move files from these legacy locations to the new config directory.
-    update_pathname( "legacy_options", "data/options.txt" );
-    update_pathname( "legacy_options2", FILENAMES["config_dir"] + "options.txt" );
-    update_pathname( "legacy_keymap", "data/keymap.txt" );
-    update_pathname( "legacy_autopickup", "data/auto_pickup.txt" );
-    update_pathname( "legacy_autopickup2", FILENAMES["config_dir"] + "auto_pickup.txt" );
-    update_pathname( "legacy_fontdata", FILENAMES["datadir"] + "fontdata.json" );
-    update_pathname( "legacy_worldoptions", "worldoptions.txt" );
-#if defined(TILES)
-    // Default tileset config file.
-    update_pathname( "tileset-conf", "tileset.txt" );
-#endif
-#if defined(SDL_SOUND)
-    // Default soundpack config file.
-    update_pathname( "soundpack-conf", "soundpack.txt" );
+    config_dir_value = user_dir_value + "config/";
 #endif
+    options_value = config_dir_value + "options.json";
+    keymap_value = config_dir_value + "keymap.txt";
+    autopickup_value = config_dir_value + "auto_pickup.json";
 }
 
-std::string PATH_INFO::find_translated_file( const std::string &pathid,
-        const std::string &extension, const std::string &fallbackid )
+std::string find_translated_file( const std::string &path, const std::string &extension,
+                                  const std::string &fallback )
 {
-    const std::string base_path = FILENAMES[pathid];
+    const std::string base_path = path;
```

Comment:
clang tidy is complaining about this not being a reference.  Seems to me that we don't even need the variable though, how about just changing the name of the parameter to `base_path`?